modules/core/src/webgl/classes/texture-formats.js

```diff
@@ -108,16 +211,31 @@ export const TEXTURE_FORMATS = {
   // [GL.COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL]: {compressed: true, gl1: ETC1}
 };
 
+export const TYPE_SIZES = {
+  [GL.FLOAT]: 4,
+  [GL.UNSIGNED_INT]: 4,
+  [GL.INT]: 4,
+  [GL.UNSIGNED_SHORT]: 2,
+  [GL.SHORT]: 2,
+  [GL.HALF_FLOAT]: 2,
+  [GL.BYTE]: 1,
+  [GL.UNSIGNED_BYTE]: 1
+};
+
 export function isFormatSupported(gl, format) {
   const info = TEXTURE_FORMATS[format];
   if (!info) {
     return false;
   }
+  if (!info.types.length === 0) {
+    // Renderbuffer-only format
+    return false;
+  }
   if (info.gl1 === undefined && info.gl2 === undefined) {
     // No info - always supported
     return true;
   }
-  const value = isWebGL2(gl) ? info.gl2 || info.gl1 : info.gl1;
+  const value = isWebGL2(gl) ? info.gl2 : info.gl1;
```

Comment:
I believe this was written this way to save space and not duplicate gl2 entries if gl1 was identical. Clearly there should have been comments indicating this.

Reply:
The issue I had was formats that exist in WebGL 1 but not 2, mainly the depth texture extension formats.